docs/users-guide/05-visualizing-results.md

```diff
@@ -140,7 +140,7 @@ Another useful option for line, area, bar, and scatter charts is trend lines. If
 
 ![Trend lines](images/visualizations/trend-lines.png)
 
-#### Line + Bar charts
+#### Line Plus Bar charts
```

Comment:
I think it would be better to leave the name, since that's the name in the system. You should be able to do something like this:
```
#### Line + Bar charts {#line-plus-bar-charts}
```

Reply:
Oh, good call, thanks!